e2e-tests/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2021"
 ark-bdk-wallet = { path = "../ark-bdk-wallet" }
 ark-client = { path = "../ark-client" }
 ark-core = { path = "../ark-core" }
+ark-grpc = { path = "../ark-grpc" }
```

Comment:
👍 To be able to use the admin client API. But I believe this is not being used directly in the e2e tests yet.